src/main/fc/rc.c

```diff
@@ -807,11 +852,19 @@ void initRcProcessing(void)
         break;
     }
 
-    for (int i = 0; i < 3; i++) {
+#ifdef USE_FEEDFORWARD
+    getFeedforwardAveraging(&pidRuntime);
```

Comment:
```suggestion
    feedforwardAveraging = pidRuntime.feedforwardAveraging;
```

Reply:
Thanks, I couldn't figure that out!  Done.